.github/workflows/catalogd-demo.yaml

```diff
@@ -20,5 +20,12 @@ jobs:
         with:
           go-version-file: "go.mod"
       - name: Run Demo Update
-        run: make demo-update
+        run: |
+          env -i \
+            HOME="$HOME" \
+            PATH="$PATH" \
+            TERM="xterm-256color" \
+            SHELL="/bin/bash" \
+            PWD="$PWD" \
```

Comment:
Are all these needed? Can you explain how this fixes the problem, when most of these are just setting the same value.

Reply:
(Yes, I know `env -i` starts with an empty environment. Do you know what's in the existing CI's environment that's causing a problem?)